### ISSUE TITLE
[RFE] Add a needinfo subcommand

### ISSUE THREAD

**Issue by JacobCallahan** [SatelliteQE/bugwrangler#1]
It would be great to be able to get a list of all active needinfos for a user or list of users. A --thin option could be used to just print out the list of links.

Examples
---------
```
$ bugwrangler needinfo --user jcallaha@redhat.com
+-----------------------------------------------------+---------------------------+
|                        link                         |          summary          |
+-----------------------------------------------------+---------------------------+
| https://bugzilla.redhat.com/show_bug.cgi?id=1177766 | [RFE] Republish composite |
| https://bugzilla.redhat.com/show_bug.cgi?id=1449011 | [RFE] Ansible integration |
+-----------------------------------------------------+---------------------------+
```
multi-user

```
+---------------------+-----------------------------------------------------+---------------------------+
|        user         |                        link                         |          summary          |
+---------------------+-----------------------------------------------------+---------------------------+
| jcallaha@redhat.com | https://bugzilla.redhat.com/show_bug.cgi?id=1177766 | [RFE] Republish composite |
| jcallaha@redhat.com | https://bugzilla.redhat.com/show_bug.cgi?id=1449011 | [RFE] Ansible integration |
| omaciel@redhat.com  | https://bugzilla.redhat.com/show_bug.cgi?id=1177723 | A totally real bug that h |
| omaciel@redhat.com  | https://bugzilla.redhat.com/show_bug.cgi?id=1449024 | Just some other bug thing |
+---------------------+-----------------------------------------------------+---------------------------+
```
multi-user thin
```
$ bugwrangler needinfo --user jcallaha@redhat.com omaciel@redhat.com rplevka@redhat.com
jcallaha@redhat.com
https://bugzilla.redhat.com/show_bug.cgi?id=12345
https://bugzilla.redhat.com/show_bug.cgi?id=12346
omaciel@redhat.com
https://bugzilla.redhat.com/show_bug.cgi?id=12347
rplevka@redhat.com
https://bugzilla.redhat.com/show_bug.cgi?id=12348
https://bugzilla.redhat.com/show_bug.cgi?id=12349
https://bugzilla.redhat.com/show_bug.cgi?id=12350
```

**Comment by omaciel** [comment]
Interesting issue. Right now one can search for `flags contain string needsinfo?` which returns the following:

```python
{'requestee': 'foo@redhat.com', 'status': '?', 'name': 'needinfo', 'modification_date': <DateTime '20180403T08:45:33' at 0x10da1a5c0>, 'creation_date': <DateTime '20180403T08:45:33' at 0x10da1a5f8>, 'is_active': 1, 'type_id': 16, 'id': 3465337, 'setter': 'bar@redhat.com'}, {'modification_date': <DateTime '20120423T19:49:34' at 0x10da1a6d8>, 'is_active': 1, 'creation_date': <DateTime '20120423T19:49:34' at 0x10da1a710>, 'status': '+', 'type_id': 12, 'name': 'screened', 'id': 1033917, 'setter': 'baz@redhat.com'}]
```

In addition I also tried to search for `flag contains string foo@redhat.com` but that does not work.

One solution would be perform the original query, and then loop through this (probably massive) list, filtering for `name=needsinfo`, `status=+` and `requestee=foo@redhat.com`.

**Comment by omaciel** [comment]
Fixed by 9249bc0

**Comment by JacobCallahan** [comment]
That is perfect!
